aiohttp/web_app.py

```diff
@@ -250,8 +251,10 @@ def add_domain(self, domain: str,
         factory = partial(MatchedSubAppResource, rule, subapp)
         return self._add_subapp(factory, subapp)
 
-    def add_routes(self, routes: Iterable[AbstractRouteDef]) -> None:
-        self.router.add_routes(routes)
+    def add_routes(
+            self, routes: Iterable[AbstractRouteDef]
+    ) -> List[Optional[AbstractRoute]]:
```

Comment:
`Optional` is bad here. The return type should be a strict list.
```suggestion
    ) -> List[AbstractRoute]:
```